build.gradle

```diff
@@ -12,7 +12,7 @@ ext {
     }
 
     if (!project.rootProject.hasProperty('vantiqSdkVersion')) {
-        vantiqSdkVersion = '1.2.4'
+        vantiqSdkVersion = '1.2.5'
```

Comment:
Seems like this should move to 1.3.0 since we're adding an entry point.  I don't think anyone's using these much, but...

Reply:
Seemed like a minor addition, not really an API change.  I've already published as `1.2.5`, so unless there are strong objections, I'm going to leave as is.

Reply:
ok